uiowa/uiowa
# Feature: brand.uiowa.edu style updates

# How to test

`blt frontend` 

**Font Awesome update to resolve https://github.com/uiowa/uids/issues/111**
- Verify that new import statement using the local libraries css works: https://github.com/uiowa/uiowa/pull/1472/commits/4422232ff907bf6373a6e98c17ddb834ca3fc707

**UIDS Version update**
- Contains the updated alert styles.   Test out new alert styles within both https://brand.local.drupal.uiowa.edu/ and https://uiowa.local.drupal.uiowa.edu/.  Let me know if you see any issues.

**brand.uiowa.edu**
- Verify that all checkboxes listed on https://github.com/uiowa/uiowa/issues/1405 have been implemented and also verify that the cards on https://uiowa.local.drupal.uiowa.edu/ still work. 
- Verify that all lockup generator style changes have been made that are listed on https://github.com/uiowa/uiowa/issues/1459
- Lockup generator pages: https://brand.local.drupal.uiowa.edu/node/add/lockup, https://brand.local.drupal.uiowa.edu/lockup-system, https://brand.local.drupal.uiowa.edu/lockup/801

**Bug Fixes**

- Go to any page on https://brand.local.drupal.uiowa.edu/web with Safari and verify that #1455 is not happening, this should resolve #1455. 

**IE 11 Fixes**

Using IE 11, Quick links nav should appear in toggle navigation:

<img width="1486" alt="Screen Shot 2020-05-01 at 12 02 05 PM" src="https://user-images.githubusercontent.com/1036433/80824444-14ea0580-8ba4-11ea-8018-ed0acb6fe98a.png">

Using IE 11, when screen width is around 800 pixels, header should still function normally with menu position:

<img width="1282" alt="Screen Shot 2020-05-01 at 12 02 21 PM" src="https://user-images.githubusercontent.com/1036433/80824491-26331200-8ba4-11ea-950d-0aabd7308227.png">

Using IE 11, cards should appear normal and not overflow at all:

<img width="1264" alt="Screen Shot 2020-05-01 at 12 02 48 PM" src="https://user-images.githubusercontent.com/1036433/80824567-4ebb0c00-8ba4-11ea-8b38-3c7ebbe985f7.png">

<img width="1379" alt="Screen Shot 2020-05-01 at 12 05 08 PM" src="https://user-images.githubusercontent.com/1036433/80824801-c721cd00-8ba4-11ea-8599-f28e6a70aff0.png">

<img width="1411" alt="Screen Shot 2020-05-01 at 12 05 19 PM" src="https://user-images.githubusercontent.com/1036433/80824831-d30d8f00-8ba4-11ea-819e-7dfe92ed15f8.png">
